.github/workflows/e2e-tests.yml

```diff
@@ -0,0 +1,51 @@
+name: Playwright Tests
+on:
+  push:
+    branches: [feat/refresh]
+  pull_request:
+    branches: [feat/refresh]
+jobs:
+  e2e_tests:
+    timeout-minutes: 60
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: postgres
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: postgres
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        ports:
+          - 5432:5432
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18
+      - name: Install dependencies
+        run: npm ci
+      - name: Copy env
+        run: cp .env.example .env
```

Comment:
I thought this needs to be `.env.local`?

Reply:
It's `.env.example` in Documenso, right?

Reply:
Yeah but for the actual env file isn't it supposed to be `.env.local`? Unless this is for prisma

https://github.com/documenso/documenso/blob/feat/refresh/apps/web/next.config.js#L6

Reply:
Tests file cause they can't access the env vars if I change it to `.env.local`